app/templates/buyers/start_brief_info.html

```diff
@@ -72,15 +80,15 @@
         <h2>Evaluate suppliers</h2>
         <ol class="list-number" start="6">
             <li>Shortlist and evaluate {{ "specialists’" if lot.slug == "digital-specialists" else "supplier" }} applications.</li>
-            <li>Award a contract to the specialist that best meets your needs.</li>
+            <li>Award a contract to the {{ "specialist’" if lot.slug == "digital-specialists" else "supplier" }} that best meets your needs.</li>
         </ol>
 
-        <p>The buying process should take around 2 weeks.</p>
+        <p>The buying process should take around 4 weeks.</p>
```

Comment:
Yes - discussed at length with Sarah at the time and it's to include the entire process until work starts, so 2 weeks for briefs open but about 4 weeks to award.